snap/snapcraft.yaml

```diff
@@ -0,0 +1,73 @@
+name: wezterm # you probably want to 'snapcraft register <name>'
+base: core22 # the base snap is the execution environment for this snap
+adopt-info: wezterm
+
+grade: stable
+confinement: classic
+
+apps:
+  wezterm:
+    command: bin/wezterm
+    common-id: org.wezfurlong.wezterm.desktop
+    desktop: assets/wezterm.desktop
+
+parts:
+  rust-deps:
+    plugin: nil
+    build-packages:
+      - curl
+    override-pull: |
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+  wezterm:
+    plugin: rust
+    source: .
+    parse-info: [assets/wezterm.appdata.xml]
```

Comment:
will this populate metadata such as the screenshots and other links that are shown from the flatpak/appimage? If not, do those need to be added here?

Reply:
And speaking of additional metadata, could you transcribe https://github.com/wez/wezterm/blob/main/.github/FUNDING.yml into appropriate `donation` entries?

Reply:
> will this populate metadata such as the screenshots and other links that are shown from the flatpak/appimage?

I actually don't know about the screenshots; we might have to see once the snap gets into the store? It's quite possible that they are

Added the `donation:` stanza.